test/include/dlaf_test/matrix/util_generic_blas.h

```diff
@@ -23,7 +23,7 @@ namespace dlaf {
 namespace matrix {
 namespace test {
 
-using namespace dlaf_test;
+using namespace dlaf::test;
```

Comment:
`using namespace` in a header is somewhat surprising (dagerous?) : https://stackoverflow.com/questions/5849457/using-namespace-in-c-headers

Reply:
Correct!